src/dali/plugin/input/rest/kucoin_futures.py

```diff
@@ -0,0 +1,288 @@
+# Copyright 2022 topcoderasdf
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
```

Comment:
Can you add API documentation links? See: https://github.com/eprbell/dali-rp2/blob/main/src/dali/plugin/input/rest/binance_com.py#L15

Reply:
fixed